Temperature-Converter-JS/assets/app.js

```diff
@@ -0,0 +1,39 @@
+const celciusInput = document.querySelector("#celcius > input");
+const fahreinheitInput = document.querySelector("#fahreinheit > input");
+const kelvinInput = document.querySelector("#kelvin > input");
+
+function roundNum(num) {
+  return Math.round(num * 100) / 100;
+}
+
+function celciusToFahAndKelv() {
+  const cTemp = parseFloat(celciusInput.value);
+  const fTemp = cTemp * (9 / 5) + 32;
+  const kTemp = cTemp + 273.15;
+  fahreinheitInput.value = roundNum(fTemp);
+  kelvinInput.value = roundNum(kTemp);
+}
```

Comment:
declare this function in the outer scope to use it in all functions 
`const pipe = (funA, funB) => (arg) => funB(funA(arg));`
then here 
```
const celciusToFahAndKelv = () => {
    const fTemp = (cTemp) => cTemp * (9 / 5) + 32;
    const kTemp = (cTemp)=> cTemp + 273.15;
    fahreinheitInput.value = pipe(parseFloat,fTemp , roundNum)(celciusInput.value);
    kelvinInput.value = pipe(parseFloat,kTemp, roundNum)(celciusInput.value)
}

```